Jarolift_MQTT.ino

```diff
@@ -44,6 +44,7 @@
 #include <EEPROM.h>
 #include <ESP8266WebServer.h>
 #include <SPI.h>
+#include <FS.h>
```

Comment:
Can you please rebase this commit ? Sqash it with with the previous commit into one?